internal/settings/settings.go

```diff
@@ -32,6 +33,11 @@ var supportedValidationPolicyReason = []string{
 type Settings struct {
 	Variables   []Variable   `json:"variables"`
 	Validations []Validation `json:"validations"`
+	/// FailurePolicy defines how the policy will response to  runtime errors and
+	// invalid or mis-configured policy definitions
+	FailurePolicy admissionregistration.FailurePolicyType `json:"failurePolicy,omitempty"`
```

Comment:
This means that the cel-policy has:

- `spec.failurePolicy` (as all Kubewarden policies): Behaviour when there's a Kubernetes Webhook error or timeout, or for matchConditions evaluation.
- `spec.settings.failurePolicy`: Behaviour for when CEL expression evaluates to false, there's CEL runtime errors, or there's invalid or mis-configured CEL. For example, mising parameters, missing variables, or a CEL expression returning false.


They are quite similar, which can be confusing. I'm thinking if we should rename `settings.failurePolicy` to `settings.CELfailurePolicy` or similar.